modules/users/client/components/forgot-password.component.ts

```diff
@@ -0,0 +1,36 @@
+import { Component } from '@angular/core';
+import { NotificationsService } from 'angular2-notifications';
+
+import { UserService } from '../services/user.service';
+
+@Component({
+  selector: 'forgot-password',
+  templateUrl: '../views/forgot-password.view.html'
+})
+export class ForgotPasswordComponent {
+  private email: string;
+
+  constructor(
+    private userService: UserService,
+    private notificationsService: NotificationsService
+  ) {
+
+  }
+
+  requestResetLink() {
+    this.userService.requestResetPasswordEmail(this.email).subscribe((data) => {
+      this.notificationsService.success('Email Sent', 'Check your inbox and follow the link in the email');
```

Comment:
Should this just be a notification?  I was thinking this was going to be a page that displayed this.  Or maybe a notification and redirect to the login screen?